hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -171,7 +171,13 @@
 {{ block.super }}
 
 {# put resource specific js below #}
-<script type="text/javascript">
+<script>
+    $(document).ready(function() {
```

Comment:
Is the option to update netcdf file will appear when a relevant metadata gets updated/created without page reload?